packages/blockchain-wallet-v4-frontend/src/components/SupportChat/components/SupportChatForGoldUserOnly/SupportChatForGoldUserOnly.tsx

```diff
@@ -58,10 +58,15 @@ export const SupportChatForGoldUserOnly: SupportChatForGoldUserOnlyComponent = (
 
   useEffect(() => {
     if (isGoldUser && userData && isAvailable) {
-      sendMessage('showChat', {
-        email: userData.email,
-        fullName: userFullname
-      })
+      // FIXME: adding a 7 second delay because sendMessage was preiously firing before react was loaded
+      // in the iframe and the chat button wasn't showing up. The proper solution should wait for the iframe
+      // to tell wallet that it's loaded properly and will receive this message.
+      setTimeout(() => {
+        sendMessage('showChat', {
+          email: userData.email,
+          fullName: userFullname
+        })
+      }, 7000)
```

Comment:
7k ms was the smallest amount of time I tried that would make the button actually show up correctly in development

Reply:
My name is saad hussien i want to withdraw my money out of my payment mobile service Egypt Vodafone cash